src/server/authserver/Main.cpp

```diff
@@ -65,31 +79,71 @@ class AuthServerSignalHandler : public acore::SignalHandler
 /// Print out the usage string for this program on the console.
 void usage(const char* prog)
 {
-    sLog->outString("Usage: \n %s [<options>]\n"
-                    "    -c config_file           use config_file as configuration file\n\r",
-                    prog);
+    printf("Usage:\n");
+    printf(" %s [<options>]\n", prog);
+    printf("    -c config_file           use config_file as configuration file\n");
+#ifdef _WIN32
+    printf("    Running as service functions:\n");
+    printf("    --service                run as service\n");
+    printf("    -s install               install service\n");
+    printf("    -s uninstall             uninstall service\n");
+#endif
 }
 
 /// Launch the auth server
 extern int main(int argc, char** argv)
 {
     // Command line parsing to get the configuration file name
     char const* configFile = _ACORE_REALM_CONFIG;
-    int count = 1;
-    while (count < argc)
+    int c = 1;
+    while (c < argc)
```

Comment:
why renaming `count` to `c` ? I'd prefer using more descriptive names for variables

Reply:
I have used the code in src/server/worldserver/Main.cpp as reference, and in it c is the var name used.

Reply:
Change done